### PR TITLE
Rename getCompleteDependencies --> buildCompleteGraph

### PR DESCRIPTION
**Pull request by elharo**
@suztomo

### CODE REVIEW

File: dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java [hunk 1]
```diff
@@ -232,11 +232,13 @@ public DependencyGraphResult getStaticLinkageCheckDependencyGraph(List<Artifact>
    * Finds the full compile time, transitive dependency graph including duplicates and conflicting
    * versions.
    */
-  public DependencyGraphResult getCompleteDependencies(Artifact artifact)
+  public DependencyGraphResult buildCompleteGraph(Artifact dependency)
```
Comment (suztomo) [inline dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java]:
```suggestion
  public DependencyGraphResult buildCompleteGraph(Artifact artifact)
```

This is artifact.

File: dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java [hunk 2]
```diff
@@ -232,11 +232,13 @@ public DependencyGraphResult getStaticLinkageCheckDependencyGraph(List<Artifact>
...
       throws RepositoryException {
 
     // root node
-    DependencyNode node = resolveCompileTimeDependencies(new DefaultDependencyNode(artifact));
+    // todo this should depend on the scope of the dependency
```
Comment (suztomo) [inline dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java]: This is a bit tricky. The root node does not have scope or optional.

File: dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraph.java [hunk 3]
```diff
@@ -34,10 +34,11 @@
 import com.google.common.collect.TreeMultimap;
 
 /**
- * A representation of the complete non-cyclic transitive dependency tree of a Maven artifact.
```
Comment (suztomo) [inline dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraph.java]: Nice. A tree does not have a cycle. "non-cyclic tree" was redundant.

File: dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java (hunk 1 above)
Comment (elharo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java]: now a dependency

**Pull request merged**